apps/files/lib/Command/RepairTree.php

```diff
@@ -26,11 +27,16 @@ protected function configure(): void {
 		$this
 			->setName('files:repair-tree')
 			->setDescription('Try and repair malformed filesystem tree structures (may be necessary to run multiple times for nested malformations)')
-			->addOption('dry-run');
+			->addOption('dry-run')
+			->addOption('storage-id', 's', InputOption::VALUE_OPTIONAL, 'If set, only repair files within the given storage ID', null)
```

Comment:
```suggestion
			->addOption('storage-id', 's', InputOption::VALUE_OPTIONAL, 'If set, only repair files within the given storage numeric ID', null)
```
If I got it correctly this is the `numeric_id`?
At least it’s called numeric_id in DB and in the output of `occ info:storages`.
Are there other ways to know the storage ID?